rest/data/host_create.go

```diff
@@ -422,6 +432,15 @@ func makeEC2IntentHost(taskID, userID, publicKey string, createHost apimodels.Cr
 		return nil, errors.Wrap(err, "inserting intent host")
 	}
 
+	appCtx, _ := env.Context()
```

Comment:
This is repeated 4 times... does it make sense to have a function to create and enqueue provisioning-create-host jobs or is that premature deduplication?

Reply:
done